docs/getting-started.md

```diff
@@ -97,41 +109,54 @@ You must have Windows Subsystem for Linux installed. To install WSL, refer to Mi
+  5. After creating an organization in the previous step, return to the CLI and run a scan:
       ```sh
-      semgrep login && semgrep ci
+     semgrep ci
       ```
 
 </TabItem>
 
 <TabItem value='Docker'>
 
-  1. Pull latest image locally:
+  1. Pull the latest image locally:
      ```sh
      docker pull returntocorp/semgrep
      ```
-   
   2. Confirm version:
       ```sh
       docker run --rm returntocorp/semgrep semgrep --version
       ```
-
-  3. Run recommended Semgrep Registry rules:
-      1. On **macOS or Linux**, in the directory to scan:
-        ```bash
-        docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep login && semgrep ci
-        ```
-        The provided `-v` option mounts the current directory into the container to be scanned. Change directories locally or provide a specific local directory in the command to scan a different directory.
-      2. On **Windows**, in the directory to scan:
+  3. For **macOS or Linux** Docker users, perform the following steps:
+     1. Sign up or log in to Semgrep Cloud Platform (SCP):
+         ```sh
+        docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep login
+         ```
+      This command automatically opens a new tab or browser window. You can also click the link in the CLI to complete the step.
+       The provided `-v` option mounts the current directory into the container to be scanned. Change directories locally or provide a specific local directory in the command to scan a different directory.
+     2. A modal appears after logging in to SCP. Click **Create an organization**. Note that you can further integrate organizations (orgs) with GitLab accounts and GitHub accounts, including personal and org accounts, after you complete this procedure.
+     3. After creating an org in the previous step, return to the CLI and run the following command to start a scan using recommended Semgrep rules: 
+         ```sh
+        docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep ci
+       ```
+  4. For **Windows** Docker users, perform the following steps: 
+     1. Sign up or log in to Semgrep Cloud Platform (SCP):
+       ```bash
+       docker run --rm -v "%cd%:/src" returntocorp/semgrep semgrep login
+       ```
+     2. A modal appears after logging in to SCP. Note that you can further integrate organizations (orgs) with GitLab accounts and GitHub accounts, including personal and org accounts, after you complete this procedure.
+     3. After creating an org in the previous step, return to the CLI and run a scan: 
         ```bash
-        docker run --rm -v "%cd%:/src" returntocorp/semgrep semgrep login && semgrep ci
+        docker run --rm -v "%cd%:/src" returntocorp/semgrep semgrep ci
         ```
```

Comment:
Add similar line here as above:

The provided `-v` option mounts the current directory into the container to be scanned. Change directories locally or provide a specific local directory in the command to scan a different directory.

Reply:
@s-santillan this still applies to guide people to understand how to use the -v option properly